packer/vhd-image-builder.json

```
@@ -69,6 +69,11 @@
             "source": "packer/install-dependencies.sh",
             "destination": "/home/packer/install-dependencies.sh"
         },
+        {
+            "type": "file",
+            "source": "parts/k8s/cloud-init/artifacts/sysctl-d-60-CIS.conf",
+            "destination": "/home/packer/sysctl-d-60-CIS.conf"
```

no need to use cp, you can use this destination interface to pave the file where you want it to go, so here:

```suggestion
            "destination": "/etc/sysctl.d/60-CIS.conf"
```

---

https://www.packer.io/docs/provisioners/file.html

---

"The file Packer provisioner uploads files to machines built by Packer. The recommended usage of the file provisioner is to use it to upload files, and then use shell provisioner to move them to the proper place, set permissions, etc."

---

so chould we make the destination change or no? sounds like we keep the `chmod` statement in the install script

---

I think you can upload it to the right place right away since /etc/ already exists. Worth trying at least. And then set permissions in the function added in #1031.

---

Done, VHD CI test should validate if that worked, as the `chmod 644 /etc/sysctl.d/60-CIS.conf` statement in `applyCIS` fail if the file doesn't exist.